src/modeler/AutoRest.Swagger/OperationBuilder.cs

```diff
@@ -51,14 +50,13 @@ public Method BuildMethod(HttpMethod httpMethod, string url, string methodName,
         {
             EnsureUniqueMethodName(methodName, methodGroup);
 
-            var method = New<Method>(new 
-            {
+            var method = New<Method>(new {
```

Comment:
While personally, I'd prefer using the `1TBS`, the rest of the project uses the traditional ANSI-style braces, and I'd prefer to keep code changes in a single brace format. 

If you could tweak the braces to be consistent that'd be great.